### PR TITLE
Add AnkerSlicer.app v0.7.5

### PR DESCRIPTION
**Pull request by monkey-debugger**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/ankerslicer.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  sha256 :no_check
+
+  on_intel do
+    url "https://public-make-moat-us.s3.us-east-2.amazonaws.com/ankermake/slicer/AnkerMake_Installer_Mac_Intel.dmg",
+        verified: "public-make-moat-us.s3.us-east-2.amazonaws.com/ankermake/slicer/"
+  end
+  on_arm do
+    url "https://public-make-moat-us.s3.us-east-2.amazonaws.com/ankermake/slicer/AnkerMake_Installer_Mac_M1.dmg",
+        verified: "public-make-moat-us.s3.us-east-2.amazonaws.com/ankermake/slicer"
+  end
```
Comment (SMillerDev) [inline Casks/ankerslicer.rb:12]:
```suggestion
  arch intel: "Intel", arm: "M1"
  version "2.4.3"
  sha256 :no_check

  url "https://public-make-moat-us.s3.us-east-2.amazonaws.com/ankermake/slicer/AnkerMake_Installer_Mac_#{arch}.dmg",
        verified: "public-make-moat-us.s3.us-east-2.amazonaws.com/ankermake/slicer/"
```

File: Casks/ankerslicer.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  name "ankerslicer"
+  desc "Slicer for AnkerMake 3D printers"
+  homepage "https://www.ankermake.com/software"
+
+  app "AnkerSlicer.app"
+end
```
Comment (SMillerDev) [inline Casks/ankerslicer.rb:19]: Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

**Review by SMillerDev — COMMENTED** [review]
This is missing a way to check if this version is the latest using a `livecheck`

**Comment by monkey-debugger** [comment]
> This is missing a way to check if this version is the latest using a `livecheck`

I'm afraid they don't have the current app version posted anywhere as of yet. `2.4.3` came from MacUpdater, but seems like they got it mistaken for the AnkerWork application. Newest version should be `0.7.5`, but can't find it anywhere but within the application itself.

I'll update the PR to use the correct version, but not sure what to do with the livecheck.

File: Casks/ankerslicer.rb (hunk 2 above)
Comment (monkey-debugger) [reply Casks/ankerslicer.rb:20]: Added zap stanza, see above

**Comment by p-linnane** [comment]
The application itself contains `2.0.0` as the version, so I've updated it to that. I was also not able to find any version information externally, so I set up the `:extract_plist` strategy to pull it from the application.

**Review by p-linnane — APPROVED** [review]
Thank you @monkey-debugger.

**Pull request merged**